### DISCUSSION TITLE
One Node Values appear in following Node in Control Panel

### DISCUSSION THREAD

**Discussion by GrizzlyAK** [zwave-js/zwave-js-ui#2030] [General]
I ran across something very odd the other day. Never noticed it before, and a page refresh solved the problem, but that didn't explain how it happened in the first place. Might point to a larger issue and wanted to bring it up here first.

I have two devices, Node 95, and Ecolink TBH500 Smart Thermostat, which has values specified under Multilevel Sensor and Thermostat Operating State. In this instance, I also noticed that in the following Node (number 96), a Zooz ZEN17 Universal Relay, that it _also_ listed a Multilevel Sensor and a Thermostat Operating State. These caught my attention and upon further inspection, confirmed that in both Nodes, the values in each of the sections was identical. I've included two images below that show Node 95 and Node 96 as they appeared before the page refresh. Note that in the Multilevel Sensor and Thermostat Operation State sections in Node 96, that the values clearly show that they belong to Node 95 (i.e., the **(95-66-0-state) Operating State**). Note in these images, the Node IDs were sorted in _descending_ order in the Control Panel.

![Capture3](https://user-images.githubusercontent.com/4411963/144950490-b3494721-9eb1-4a89-af0e-24bb746a8742.JPG)

![Capture2](https://user-images.githubusercontent.com/4411963/144950505-3ac1181c-bbe7-47a4-a01d-619d273c52fc.JPG)

**Comment by robertsLando** [comment]
Oh maybe I know what's going on here, thanks for your issue @GrizzlyAK

**Reply by GrizzlyAK** [reply]
You're welcome. And thank YOU for zwavejs2mqtt!

**Comment by robertsLando** [comment]
Could you tell me when this happens? I mean after adding/removing a node? Or randomly? Is there a specific order of operations that gives that state?

**Reply by GrizzlyAK** [reply]
Yeah, unfortunately, Daniel, I've only noticed it this one time. Both of these nodes had been added for a while, and I think I was probably adding some others 101, 102  perhaps, and was checking config params when I noticed it. I just happened to see that the Relay device had a Thermostat value. That's what caught my attention. Like I said, a page refresh cleared it up. The additional values seemed to be labeled correctly, just duplicated in the wrong block. I'll keep an eye out for similar cases and let you know if I find anything and try to capture more details. Sorry. That's why I didn't list it as an issue, since I couldn't reproduce it.
